GEOSldas_App/util/postproc/ObsFcstAna_stats/Plot_stats_maps.py

```diff
@@ -171,6 +138,8 @@ def plot_OmF_maps(postproc_obj, stats, fig_path='./'):
 # -----------------------------------------------------------------------------------------------
     
 if __name__ == '__main__':
+
+    # Plot maps of long-term temporal stats
```

Comment:
I deleted the application of the Nmin threshold from compute_temporal_stats_from_sums().  The Nmin=20 value was far too hidden in the depths of the calculation there.  The threshold still needs to be applied here, but I didn't implement it.  First, we need to find a good place for the specification of the Nmin value.   It could be put into user_config.py, but since Nmin can be applied at the very end (when stats are plotted), we could also put it into the main() function in Plot_stats_maps.py.   More generally, we need to settle on where exactly we want the user to edit the scripts' parameters.